src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -164,18 +164,18 @@ Dashboards features include:
 >
     You can customize the color of your charts per series in area, bar, line, pie, and stacked bar charts.
   
-   To customize the color per series:
+    To customize the color per series:
     1. Run your query.
     1. Open the Colors menu in the chart settings.
     1. Select the plus <Icon name="fe-plus"/> icon.
     1. Select the series you want to change the color.
     1. Select the desired color.
```

Comment:
Should we change this to consecutive numbers?

Reply:
Yes, I'm going to change it. Thanks!